homework/index.js

```diff
@@ -29,22 +31,115 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function addRepoDetailsInfo(listElm, title, value) {
+    const trElm = createAndAppend('tr', listElm, { class: 'tr-elm' });
+    createAndAppend('td', trElm, {
+      class: 'titles',
+      text: title,
+    });
+
+    createAndAppend('td', trElm, {
+      class: 'values',
+      text: value !== null ? value : 'No Information',
+    });
+
+    return trElm;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repos) => {
-      const root = document.getElementById('root');
-      if (err) {
+  function formatDate(stringDateAndTime) {
```

Comment:
This escaped my eye in the previous review. This function does not take a string as an argument. Instead, it expects a Date object. Therefore this argument is best named simply `date`. I keep stressing on using accurate and precise names because your code becomes confusing if the names do not accurately reflect the data they represent.

Reply:
thank you for the corrections. I just thought,I have to use string in my parameter when I use toLocaleString. now it is clear for me. I made it  _date_.